atlp-rwanda/space-bn-frontend
# #175743423 ft(notifications): mark notifications as read #175743423

#### What does this PR do?

- Mark notifications as read

#### Description of Task to be completed? 

- Get all  notifications with status unread and mark as read

#### How should this be manually tested?

- Clone the repo 

- In your home directory  cd this branch **`ft-mark-notifications-as-read-#175743423`**

- Then run **`npm install`** to install all dependencies

- Run **`npm start`** to start application
- After successfully logged in you created a request in postman 
- Then go login and go in dashboard
- Click on `bell icon` after dialog displayed just click on the double check button on the right corner of dialog 
- Then you should see a toaster notifying that `All notifications marked as read`


#### Any background context you want to provide?

- User can only read his/her notifications
- You need to login to be able to see your notifications and mark them as read
- Make sure your backend is started
- In your `.env` add this variable **`REACT_APP_PROXY`** and the value will be where your server is running
#### What are the relevant Pivotal Tracker stories?

- [#175743423 ](https://www.pivotaltracker.com/n/projects/2476677/stories/175743423)

#### Screenshots (if appropriate)

- N/A

#### Questions:

- N/A
